setup-fake-timers.ts

```diff
@@ -0,0 +1,7 @@
+import fakeTimers from '@sinonjs/fake-timers';
+
+const clock = fakeTimers.install({
+  toFake: Object.keys(fakeTimers.timers) as fakeTimers.FakeMethod[],
+});
+
+export { clock };
```

Comment:
```suggestion
export const clock = fakeTimers.install({
  toFake: Object.keys(fakeTimers.timers) as fakeTimers.FakeMethod[],
});
```

Reply:
Good! Honestly, I just copied this code from another project :D